### PR TITLE
update nodejs variables

### PR DESCRIPTION
**Pull request by kayiwa**
the new nodejs role downloads the binary version of the file. We add the
values matched on the vms

### CODE REVIEW

**Comment by kayiwa** [comment]
this will close #2828

File: group_vars/abid/production.yml [hunk 1]
```diff
@@ -12,8 +12,7 @@ passenger_app_env: "production"
 passenger_ruby: "/usr/bin/ruby2.6"
 ruby_version_override: "ruby2.6"
 bundler_version: "2.2.19"
-nodejs__upstream_release: 'node_12.x'
-nodejs__upstream_key_id: '68576280'
+desired_nodejs_version: "v12.22.6"
```
Comment (jrgriffiniii) [inline group_vars/abid/production.yml:15]: This is not a blocking criticism, however I am curious as to why the minor and patch versions are not tracking the latest release for https://nodejs.org/download/release/latest-erbium/?

File: group_vars/dpul_staging.yml [hunk 2]
```diff
@@ -8,8 +8,7 @@ passenger_app_env: "staging"
 passenger_ruby: "/usr/bin/ruby2.6"
 ruby_version_override: "ruby2.6"
 bundler_version: "2.1.4"
-nodejs__upstream_release: 'node_12.x'
-nodejs__upstream_key_id: '68576280'
+desired_nodejs_version: "v12.22.10"
```
Comment (jrgriffiniii) [inline group_vars/dpul_staging.yml:11]: 👍

**Review by jrgriffiniii — APPROVED** [review]
Inspecting https://github.com/pulibrary/princeton_ansible/blob/main/roles/nodejs/tasks/main.yml#L34, this seems perfect to me. Thank you very much as always @kayiwa!

**Review by christinach — DISMISSED** [review]
@kayiwa can you also include approvals with node version v12.22.5 
It is what we have in the production box.

File: group_vars/abid/production.yml (hunk 1 above)
Comment (kayiwa) [reply group_vars/abid/production.yml:15]: The problem we encountered with latest is some folks preferred to pin versions and brought this up.

**Pull request merged**